[2025-03-04 00:29:44,452] Running inverse folding on PDB/CIFs ...
[2025-03-04 00:29:44,454] Will sample 5 sequences from 1 PDBs at temperature(s) [0.5] and regions: ['CDR1', 'CDR2', 'CDR3']
[2025-03-04 00:29:47,779] Loaded model to cpu.
[2025-03-04 00:29:47,786] NOTE: Custom chain mode enabled. Will run specified chain(s).
[2025-03-04 00:29:47,787] Reading in (1) PDBs from DataFrame
[2025-03-04 00:29:47,874] Predicting batch 1/1: PDBs 1-1 out of 1 total
[2025-03-04 00:29:51,470] Saving 1 CSVs to antifold_output
[2025-03-04 00:29:51,474] Writing 8ee8_imgt_C per-residue log probs CSV to antifold_output/8ee8_imgt_C.csv
[2025-03-04 00:29:51,499] Prediction encountered an unexpected error: not enough values to unpack (expected 2, got 1)
Traceback (most recent call last):
  File "/home/kalyani/Kalyani/Internship/Tamarind-tasks/AntiFold/antifold/main.py", line 488, in <module>
    main(args)
  File "/home/kalyani/Kalyani/Internship/Tamarind-tasks/AntiFold/antifold/main.py", line 438, in main
    pdb_output_dict = sample_pdbs(
  File "/home/kalyani/Kalyani/Internship/Tamarind-tasks/AntiFold/antifold/main.py", line 227, in sample_pdbs
    fasta_dict = sample_from_df_logits(
  File "/home/kalyani/miniconda3/envs/antifold_cpu/lib/python3.10/site-packages/antifold/antiscripts.py", line 663, in sample_from_df_logits
    H_orig, L_orig = get_df_seqs_HL(df_logits)
  File "/home/kalyani/miniconda3/envs/antifold_cpu/lib/python3.10/site-packages/antifold/antiscripts.py", line 647, in get_df_seqs_HL
    df_H, df_L = get_dfs_HL(df)
  File "/home/kalyani/miniconda3/envs/antifold_cpu/lib/python3.10/site-packages/antifold/antiscripts.py", line 631, in get_dfs_HL
    Hchain, Lchain = df["pdb_chain"].unique()[:2] # Assume heavy, light
ValueError: not enough values to unpack (expected 2, got 1)
